docs/supported_languages_and_lockfiles.md

```diff
@@ -70,6 +70,14 @@ Submoduled dependencies are included in the project folder retain their Git hist
 
 Vendored dependencies have been directly copied into the project folder, but do not retain their Git histories. OSV-Scanner uses OSV's [determineversion API](https://google.github.io/osv.dev/post-v1-determineversion/) to estimate each dependency's version (and associated Git Commit). Vulnerabilities for the estimated version are returned. This process requires no additional work from the user. Run OSV-Scanner as you normally would.
 
+## Transitive dependency scanning
+
+OSV-Scanner now supports transitive dependency scanning for Maven pom.xml. This feature is enabled by default when scanning, but it is disabled in the [experimental offline mode](./offline-mode.md).
```

Comment:
nit: "now" is not required for documentation. it makes more sense in the context of an announcement.